### PR TITLE
improve docs about how MNE-HCP does things

### PR DESCRIPTION
**Pull request by dengemann**
@jona-sassenhagen @jasmainak is this now clearer? Can you take a look?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -120,6 +120,35 @@ Why we are not globbing files? Because the HCP-MEG data are fixed, all file
...
+
+The HCP for MEG provides coregistration information for native BTI/4D
+setting. MNE-Python expects coordinates in meters and the neuromag
+right anterior superior (RAS) coordinates. However, essential information is
+missing to compute all transforms needed to easily perform the conversions.
+
+For now, the way things work, all processing is performed in native BTI/4D
+coordinates with the device to head transform skipped (set to identity matrix),
+such that the coregistration directly maps from the sensors to freesurfer
+space. This has a few minor consequences that you may be wondering about
```
Comment (jona-sassenhagen) [inline README.md]: 'may be surprising/confusing'?

File: README.md [hunk 2]
```diff
@@ -120,6 +120,35 @@ Why we are not globbing files? Because the HCP-MEG data are fixed, all file
...
+when being used to work with MNE-Python.
+
+1. In the reader code you will see many flags set to ```convert=False```, etc.
+This is not a bug.
+
+2. All channel names and positions are native, topographic plotting might not
```
Comment (jona-sassenhagen) [inline README.md]: Maybe split this in 2 sections:

- 2a. For sensor data, ...
- 2b. For source analyses, ...

File: README.md [hunk 3]
```diff
@@ -120,6 +120,35 @@ Why we are not globbing files? Because the HCP-MEG data are fixed, all file
 patterns are known and access via Amazon web services easier if the files
 to be accessed are known in advance.
 
+## Gotchas
```
Comment (jasmainak) [inline README.md]: nice addition :) I'll add a few comments for the typos

File: README.md [hunk 4]
```diff
@@ -120,6 +120,35 @@ Why we are not globbing files? Because the HCP-MEG data are fixed, all file
...
+
+
+### Native coordinates and resulting plotting and processing peculiartities
```
Comment (jasmainak) [inline README.md]: pecularities

File: README.md [hunk 5]
```diff
@@ -120,6 +120,35 @@ Why we are not globbing files? Because the HCP-MEG data are fixed, all file
...
+
+The HCP for MEG provides coregistration information for native BTI/4D
+setting. MNE-Python expects coordinates in meters and the neuromag
```
Comment (jasmainak) [inline README.md]: Neuromag

File: README.md [hunk 6]
```diff
@@ -120,6 +120,35 @@ Why we are not globbing files? Because the HCP-MEG data are fixed, all file
...
+right anterior superior (RAS) coordinates. However, essential information is
+missing to compute all transforms needed to easily perform the conversions.
+
+For now, the way things work, all processing is performed in native BTI/4D
+coordinates with the device to head transform skipped (set to identity matrix),
```
Comment (jasmainak) [inline README.md]: device-to-head

File: README.md [hunk 7]
```diff
@@ -120,6 +120,35 @@ Why we are not globbing files? Because the HCP-MEG data are fixed, all file
...
+such that the coregistration directly maps from the sensors to freesurfer
+space. This has a few minor consequences that you may be wondering about
+when being used to work with MNE-Python.
+
+1. In the reader code you will see many flags set to ```convert=False```, etc.
+This is not a bug.
+
+2. All channel names and positions are native, topographic plotting might not
+work as as expected. First of all the layout file is not recognized, second,
+the coordinates are not regonized as native ones, tilting the plot by 90
```
Comment (jasmainak) [inline README.md]: tilting the plot -> rotating the legend

File: README.md [hunk 8]
```diff
@@ -120,6 +120,35 @@ Why we are not globbing files? Because the HCP-MEG data are fixed, all file
...
+degree, etc. To fix this either a proper layout can be computed with
+```hcp.preprocessing.make_hcp_bti_layout``` or the conversion to MNE can be
+performed post-hoc using ```hcp.preprocessing.transform_sensors_to_mne```.
+Note that source localization will be wrong when computerd on transformed
```
Comment (jasmainak) [inline README.md]: computed

File: README.md [hunk 9]
```diff
@@ -120,6 +120,35 @@ Why we are not globbing files? Because the HCP-MEG data are fixed, all file
...
+objects. As things are coordinates have to be kept in the native space to be
```
Comment (jasmainak) [inline README.md]: As things are,

File: README.md [hunk 10]
```diff
@@ -120,6 +120,35 @@ Why we are not globbing files? Because the HCP-MEG data are fixed, all file
...
+when being used to work with MNE-Python.
+
+1. In the reader code you will see many flags set to ```convert=False```, etc.
+This is not a bug.
+
+2. All channel names and positions are native, topographic plotting might not
+work as as expected. First of all the layout file is not recognized, second,
+the coordinates are not regonized as native ones, tilting the plot by 90
+degree, etc. To fix this either a proper layout can be computed with
+```hcp.preprocessing.make_hcp_bti_layout``` or the conversion to MNE can be
```
Comment (jasmainak) [inline README.md]: actually the layout file isn't used in the joint_plot

File: README.md (hunk 3 above)
Comment (dengemann) [inline README.md]: >  typos

attentional landmark, this way I know who read it :)

File: README.md (hunk 7 above)
Comment (dengemann) [inline README.md]: but I'm saying that the layout looks rotated by 90 degrees. It's not just the legend.

Comment (dengemann) [inline README.md]: Let's rephrase more generally.

Comment (jasmainak) [inline README.md]: yes agreed. Also, regonized -> recognized.

@dengemann you have a talent for typos ;)

**Comment by dengemann** [comment]
Thanks for your feedback @jasmainak and @jona-sassenhagen !

**Pull request merged**